src/grids/css/grids-core.css

```diff
@@ -55,12 +48,9 @@
 
 .pure-u {
     display: inline-block;
-    *display: inline; /* IE < 8: fake inline-block */
-    zoom: 1;
     letter-spacing: normal;
     word-spacing: normal;
-    vertical-align: top;
-    text-rendering: auto;
+    vertical-align: bottom;
```

Comment:
@lkraav no it's not connected to text-rendering. `vertical-align` needs to go to the bottom to fix #539